components/NavigationBar.js

```diff
@@ -1,22 +1,53 @@
 import React from 'react'
-import { Navbar, Nav, NavDropdown } from 'react-bootstrap/'
+import Link from 'next/link'
+import { useRouter } from 'next/router'
+import { deleteMe } from '../utils/auth'
+import { Button, Navbar, Nav } from 'react-bootstrap/'
 
 const NavigationBar = () => {
+  const router = useRouter()
+
+  const handleSignIn = () => {
+    router.push('/api/spotify-auth')
+  }
+
+  const handleSignOut = async () => {
+    try {
+      await deleteMe()
+    } catch (error) {
+      console.log('sign out', error)
+    }
+  }
+
   return (
-    <Navbar style={{ position: 'absolute', top: '0' }} className="navbar-dark bg-dark" expand="lg">
-      <Navbar.Brand href="#home">MusicJournal</Navbar.Brand>
-      <Navbar.Toggle aria-controls="basic-navbar-nav" />
-      <Navbar.Collapse id="basic-navbar-nav">
+    <Navbar collapseOnSelect className="navbar-dark bg-dark" expand="lg">
+      <Link href="/" passHref>
```

Comment:
Is there a reason for using passHref here, or is it to leave it open for child elements to use the href property? From the [documentation](https://nextjs.org/docs/api-reference/next/link) that seems to be what it does. Not a criticism just curious on what it does